cmds/signin/signin.go

```diff
@@ -87,11 +88,11 @@ func cmdSignin(cmd *cobra.Command, _ []string) error {
 		qs := r.URL.Query()
 		csh, _ := cmd.Flags().GetBool("csh")
 		if csh {
-			fmt.Fprintln(cmd.OutOrStdout(), "setenv TASKCLUSTER_CLIENT_ID '" + qs.Get("clientId") + "'")
-			fmt.Fprintln(cmd.OutOrStdout(), "setenv TASKCLUSTER_ACCESS_TOKEN '" + qs.Get("accessToken") + "'")
+			fmt.Fprintln(cmd.OutOrStdout(), "setenv TASKCLUSTER_CLIENT_ID '"+qs.Get("clientId")+"'")
+			fmt.Fprintln(cmd.OutOrStdout(), "setenv TASKCLUSTER_ACCESS_TOKEN '"+qs.Get("accessToken")+"'")
 		} else {
-			fmt.Fprintln(cmd.OutOrStdout(), "export TASKCLUSTER_CLIENT_ID='" + qs.Get("clientId") + "'")
-			fmt.Fprintln(cmd.OutOrStdout(), "export TASKCLUSTER_ACCESS_TOKEN='" + qs.Get("accessToken") + "'")
+			fmt.Fprintln(cmd.OutOrStdout(), "export TASKCLUSTER_CLIENT_ID='"+qs.Get("clientId")+"'")
+			fmt.Fprintln(cmd.OutOrStdout(), "export TASKCLUSTER_ACCESS_TOKEN='"+qs.Get("accessToken")+"'")
```

Comment:
I guess my lint-on-save did some tricks here...

Reply:
yeah, gofmt is great until different versions of it fight with themselves.. can you revert this bit?

Reply:
I'd rather not... this is the gofmt version that tc-worker uses... and then one we should be using everywhere... I think it's just that this repo doesn't run sufficient linter tests...